_inc/client/security/loading-card.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+
+export function LoadingCard( props ) {
```

Comment:
This looks like such a generic component - I wonder if it could live in `components/` too.

Reply:
I decided it was named too generically for what it is since it only wrapped `SettingsCard`, so I just deleted it.